File: app/src/main/java/rectangledbmi/com/pittsburghrealtimetracker/world/jsonpojo/VehicleResponse.java

package rectangledbmi.com.pittsburghrealtimetracker.world.jsonpojo;

import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * First retrofit object to get vehicles
 *
 * @author Jeremy Jao
 * @since 46
 */
@Generated("org.jsonschema2pojo")
public class VehicleResponse {

    @SerializedName("bustime-response")
    @Expose
    private BustimeVehicleResponse bustimeResponse;

    /**
     *
     * @return
     *     The bustimeResponse
     */
    public BustimeVehicleResponse getBustimeResponse() {
        return bustimeResponse;
    }

    /**
     *
     * @param bustimeResponse
     *     The bustime-response
     */
    public void setBustimeResponse(BustimeVehicleResponse bustimeResponse) {
        this.bustimeResponse = bustimeResponse;
    }

}

File: pat-static/src/main/java/com/rectanglel/patstatic/vehicles/response/VehicleResponse.kt
package com.rectanglel.patstatic.vehicles.response

import javax.annotation.processing.Generated
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * First retrofit object to get vehicles
 *
 * @author Jeremy Jao
 * @since 46
 */
@Generated("org.jsonschema2pojo")
data class VehicleResponse (

    @SerializedName("bustime-response")
    @Expose
    val bustimeResponse: BustimeVehicleResponse? = null

)